client/main.lua

```diff
@@ -4,6 +4,7 @@
 
 local config = require 'config.client'
 local functions = require 'client.functions'
+local childLockEnabled = GetConvar('qbx:vehiclekeys:enablechildlock', 'true') == 'true'
```

Comment:
Why should it be the default behavior?

Reply:
should we disable it by default? (I find that for the roleplay action the default setting true is best)

Reply:
https://github.com/Qbox-project/qbx_vehiclekeys/pull/51#pullrequestreview-2081486256